src/components/Home.js

```diff
@@ -69,10 +69,10 @@ function Home() {
           Technology.
         </div>
         <div className="scroll-down"></div>
-        <img src="home1.png" className="home-imgs absolute w-1/12 bottom-20 left-20" alt="home_img" />
-        <img src="home2.png" className="home-imgs-2 absolute w-1/12 top-20 right-96 z-[1]" alt="home_img" />
-        <img src="home3.png" className="home-imgs-2 absolute w-1/12 top-32 left-96 z-[1]" alt="home_img" />
-        <img src="home4.png" className="home-imgs absolute w-[12%] bottom-0 right-[22vw] z-[1]" alt="home_img" />
+        <img src="/home1.png" className="home-imgs absolute w-1/12 bottom-20 left-20" alt="home_img" />
+        <img src="/home2.png" className="home-imgs-2 absolute w-1/12 top-20 right-96 z-[1]" alt="home_img" />
+        <img src="/home3.png" className="home-imgs-2 absolute w-1/12 top-32 left-96 z-[1]" alt="home_img" />
+        <img src="/home4.png" className="home-imgs absolute w-[12%] bottom-0 right-[22vw] z-[1]" alt="home_img" />
```

Comment:
Please remove all the forward slashes you added in the src path of all img tags.
```suggestion
        <img src="home1.png" className="home-imgs absolute w-1/12 bottom-20 left-20" alt="home_img" />
        <img src="home2.png" className="home-imgs-2 absolute w-1/12 top-20 right-96 z-[1]" alt="home_img" />
        <img src="home3.png" className="home-imgs-2 absolute w-1/12 top-32 left-96 z-[1]" alt="home_img" />
        <img src="home4.png" className="home-imgs absolute w-[12%] bottom-0 right-[22vw] z-[1]" alt="home_img" />
```

Reply:
Done